src/config.js

```diff
@@ -4,4 +4,6 @@ export default {
   compatibilityUrl: 'https://zulipchat.com/compatibility',
   defaultLoginEmail: '', // useful during development, empty for production
   defaultLoginPassword: '', // useful during development, empty for production
+  appStoreLink: 'https://itunes.apple.com/us/app/facebook/id284882215?mt=8',
```

Comment:
I assume these are placeholder links?

Probably best to set these to the empty string before checking in this diff